### ISSUE TITLE
loss problem

### ISSUE THREAD

**Issue by suxi1127** [amgantonio/SAR2NDVI_CNN#1]
Hi,thanks for your sharing your code with us! 
When I try to run your train.py, some error occurs.I found that the training loss is always 330.xxx and val loss is always 332.xxx , they do not change or drop.
Moreover, when I try to test the model which is under this condition, the output is definitely 0!
So what's the problem?Is there any wrong operation?I use your original code and dataset you provide, but the loss seems not converge and remains a large one.The only thing I changed is 

X_train, y_train, X_val, y_val = load_dataset(data_folder, ps, int(r), identifier,num)

I use int(r) instead of r because some float/int error. 
My python version is 3.5,and I use cpu to train.
Looking forward to your reply soon!!!

**Comment by suxi1127** [comment]
Oh,I also changed the data and output folder dir, and the identifier is 'SOPTII'.

**Comment by zhangzy003** [comment]
Thanks for your sharing your code with me!
When I try to run your train.py, some error occurs.I found that the training loss is too much, and it leads to the bad result.
I am looking forward to your reply!
